### ISSUE TITLE
New release? Is anybody managing this anymore?

### ISSUE THREAD

**Issue by rlam3** [thadeusb/flask-cache#151]
New release? Is anybody managing this anymore?

**Comment by wellsoliver** [comment]
Yeah, maybe we could transfer the project to an active maintainer? Little things like the move from flask.ext.cache to flask_cache on the exports...

**Comment by sh4nks** [comment]
In case you are interested, I am maintaining a Flask-Cache fork here: [flask-caching](https://github.com/sh4nks/flask-caching).

**Comment by rfinz** [comment]
@sh4nks has your version diverged very heavily from the original? it seems that @thadeusb has been absent from the internet for a while. Hope he's alright. Would be nice to reclaim the name on pypi, though. "flask-caching" is a teeny bit clunky ;P

**Comment by sh4nks** [comment]
Yeah I know it's not the best name but still better than one with a suffix ;) I hope so too.. I would be happy to abandon my fork again but as long as nothing happens here I try to maintain it.
I have documented everything that I have changed/merged into my fork: https://pythonhosted.org/Flask-Caching/changelog.html

**Comment by rlam3** [comment]
@sh4nks I would love to see Flask-Caching be updated here: http://flask.pocoo.org/extensions/ 
Or another idea is to just create a newly updated extension toolchest with most updated flask extensions. There are just many tools and not well maintained extension

**Comment by sayeghr** [comment]
I actually had the same question, sent out an email to Thadeus but I suspect I wont get a response?

**Comment by wellsoliver** [comment]
Now I'm just worrying about the dude.

**Comment by sh4nks** [comment]
@wellsoliver yeah me too, I hope he's ok!

**Comment by kaplun** [comment]
According to Facebook, he seemed to be OK at least until August :) https://www.facebook.com/thadeusb

Very likely different new priorities in life :smile:

**Comment by michaelkuty** [comment]
@thadeusb please move this project under https://github.com/jazzband/ which has qualified set of mainteners which can make some simple PRs etc and keep this project alive.
